### PR TITLE
T03 retrieve user me

### PR DESCRIPTION
**Pull request by eliosfg**
T03: Retrieve Me User test added

### CODE REVIEW

File: src/main/java/api/methods/APIUsersMethods.java [hunk 1]
```diff
@@ -1,43 +1,68 @@
...
 
         if (response.jsonPath().getString("id") == null) {
+            log.error("Failed to create user");
+            return null;
+        } else {
+            return response;
+        }
+    }
+
+    public static Response createAUser(String username) {
```
Comment (adrianovi) [inline src/main/java/api/methods/APIUsersMethods.java:41]: Do you need 2 different methods for creating users? And if so, this one needs a username parameter?

Comment (eliosfg) [reply src/main/java/api/methods/APIUsersMethods.java:41]: Yes.I was thinking a long and I needed it to create a new User with different username, generate your own authentication token and then  change the current logged in user. I couldn't find another way to do that. If you have a suggestion let me know

Comment (eliosfg) [reply src/main/java/api/methods/APIUsersMethods.java]: I just refactored it to improve my implementation.

**Review by adrianovi — APPROVED** [review]

**Review by Agustin-Mediotti — APPROVED** [review]
Good Job 🧉

**Pull request merged**